site/docs/types/user_defined_types.md

```diff
@@ -12,3 +12,5 @@ A YAML example of an extension type is below:
 ```
 
 This declares a new type (namespaced to the associated YAML file) called "point". This type is composed of two `i32` values named longitude and latitude. Once a type has been declared, it can be used in function declarations.  [TBD: should field references be allowed to dereference the components of a user defined type?]
+
+The canonical physical representation of user-defined types is a protobuf [Any](https://developers.google.com/protocol-buffers/docs/proto3#any) message.
```

Comment:
Please reword this. There is no canonical physical representation. There is a protobuf representation of literals only.